src/SprykerEco/Yves/Computop/Controller/CallbackController.php

```diff
@@ -99,13 +99,14 @@ public function successIdealAction()
      *
      * @return \Symfony\Component\HttpFoundation\RedirectResponse
      */
-    protected function successPrePlaceAction(ComputopPrePostPaymentHandlerInterface $handler)
+    protected function successPostPlaceAction(ComputopPrePostPaymentHandlerInterface $handler)
     {
         $quoteTransfer = $this->getFactory()->getQuoteClient()->getQuote();
-        $quoteTransfer = $handler->handle(
-            $quoteTransfer,
-            $this->responseArray
-        );
+        $quoteTransfer = $handler
+            ->handle(
+                $quoteTransfer,
+                $this->responseArray
+            );
 
         if ($quoteTransfer->getPayment()->getPaymentMethod() === ComputopConfig::PAYMENT_METHOD_EASY_CREDIT
```

Comment:
This code-block is not extendable.
If once in a future developer will require more validations for specific payment method(s), whole `successPostPlaceAction` should be replaced on project level.